ts/SplineTrackType.ts

```diff
@@ -71,4 +93,29 @@ export const switchExtraLegs: Partial<Record<SplineTrackType, HermiteCurve[]>> =
     [SplineTrackType.RAIL_914_SWITCH_RIGHT_MIRROR]: [secondLegSwitchRight],
     [SplineTrackType.RAIL_914_SWITCH_RIGHT_MIRROR_NOBALLAST]: [secondLegSwitchRight],
     [SplineTrackType.RAIL_914_SWITCH_RIGHT_NOBALLAST]: [secondLegSwitchRight],
+    // snow stuff
+    [SplineTrackType.RAIL_914_SWITCH_3WAY_LEFT_SNOW]: [secondLegSwitchLeft, secondLegSwitchRight],
+    [SplineTrackType.RAIL_914_SWITCH_3WAY_LEFT_NOBALLAST_SNOW]: [secondLegSwitchLeft, secondLegSwitchRight],
+    [SplineTrackType.RAIL_914_SWITCH_3WAY_RIGHT_SNOW]: [secondLegSwitchLeft, secondLegSwitchRight],
+    [SplineTrackType.RAIL_914_SWITCH_3WAY_RIGHT_NOBALLAST_SNOW]: [secondLegSwitchLeft, secondLegSwitchRight],
+    [SplineTrackType.RAIL_914_SWITCH_CROSS_45_SNOW]: [{
+        startPoint: {x: 87.9, y: 212.1, z: 0},
+        startTangent: {x: 424.2, y: -424.2, z: 0},
+        endPoint: {x: 512.1, y: -212.1, z: 0},
+        endTangent: {x: 424.2, y: -424.2, z: 0},
+    }],
+    [SplineTrackType.RAIL_914_SWITCH_CROSS_90_SNOW]: [{
+        startPoint: {x: 191.2, y: -191.2, z: 0},
+        startTangent: {x: 0, y: 382.4, z: 0},
+        endPoint: {x: 191.2, y: 191.2, z: 0},
+        endTangent: {x: 0, y: 382.4, z: 0},
+    }],
```

Comment:
nit: It would be good to create a constant for the 45/90 legs since they are reused in two places.